src/tests

```diff
@@ -3,5 +3,5 @@ p({"class":"foam.nanos.test.Test","id":"MDAO test","description":"MDAO test","se
+p({"class":"foam.dao.SequenceNumberDAOTest","id":"SequenceNumberDAO test","description":"SequenceNumberDAO tests"})r({"class":"foam.nanos.test.Test","id":"AuthenticatedJunctionDAOTest"})
```

Comment:
Did you mean to remove the AuthenticatedJunctionDAOTest?

Reply:
`r({"class":"foam.nanos.test.Test","id":"AuthenticatedJunctionDAOTest"})`

That's at the end of the line. That would remove the test wouldn't it?